Makefile

```diff
@@ -239,6 +239,9 @@ v8:
 .PHONY: test
 # This does not run tests of third-party libraries inside deps.
 test: all ## Runs default tests, linters, and builds docs.
+ifeq ($(OSTYPE)$(NODE_TARGET_TYPE),darwinshared_library)
+	$(MAKE) -s change-install-name
+endif
```

Comment:
Maybe it would be better to make `change-install-name` a no-op itself, rather than making its usage conditional? The current implementation wouldn’t work on other OSes anyway, right?

Reply:
it's only for macOS. so you mean make it no-op for other platform?

Reply:
@yhwang Yes, exactly that 👍